pkg/monitor/dynkeepalived.go

```
@@ -167,11 +196,11 @@ func handleBootstrapStopKeepalived(kubeconfigPath string, bootstrapStopKeepalive
 	}
 
 	for {
-		if _, err := config.GetIngressConfig(kubeconfigPath, []string{}, nodeCache); err != nil {
+		if !isKubeApiHealthy(apiPort) {
 			// We have started to talk to Ironic through the API VIP as well,
 			// so if Ironic is still up then we need to keep the VIP, even if
 			// the apiserver has gone down.
-			if _, err = http.Get("http://localhost:6385/v1"); err != nil {
+			if _, err := http.Get("http://localhost:6385/v1"); err != nil {
```

golangci-lint (2.12.2)

[error] response body must be closed

(bodyclose)